v2/pkg/operators/common/dsl/dsl_test.go

```diff
@@ -94,83 +94,78 @@ func TestDslFunctionSignatures(t *testing.T) {
+	generate_java_gadget(arg1, arg2, arg3 interface{}) interface{}
+	gzip(arg1 interface{}) interface{}
+	gzip_decode(arg1 interface{}) interface{}
+	hex_decode(arg1 interface{}) interface{}
+	hex_encode(arg1 interface{}) interface{}
+	hex_to_dec(arg1 interface{}) interface{}
+	hmac(arg1, arg2, arg3 interface{}) interface{}
+	html_escape(arg1 interface{}) interface{}
+	html_unescape(arg1 interface{}) interface{}
+	join(separator string, elements ...interface{}) string
+	len(arg1 interface{}) interface{}
+	line_ends_with(str string, suffix ...string) bool
+	line_starts_with(str string, prefix ...string) bool
+	md5(arg1 interface{}) interface{}
+	mmh3(arg1 interface{}) interface{}
+	oct_to_dec(arg1 interface{}) interface{}
+	print_debug(args ...interface{})
+	rand_base(length uint, optionalCharSet string) string
+	rand_char(optionalCharSet string) string
+	rand_int(optionalMin, optionalMax uint) int
+	rand_ip(cidr ...string) string
+	rand_text_alpha(length uint, optionalBadChars string) string
+	rand_text_alphanumeric(length uint, optionalBadChars string) string
+	rand_text_numeric(length uint, optionalBadNumbers string) string
+	regex(arg1, arg2 interface{}) interface{}
+	remove_bad_chars(arg1, arg2 interface{}) interface{}
+	repeat(arg1, arg2 interface{}) interface{}
+	replace(arg1, arg2, arg3 interface{}) interface{}
+	replace_regex(arg1, arg2, arg3 interface{}) interface{}
+	reverse(arg1 interface{}) interface{}
+	sha1(arg1 interface{}) interface{}
+	sha256(arg1 interface{}) interface{}
+	sha512(arg1 interface{}) interface{}
+	sort(args ...interface{}) interface{}
+	starts_with(str string, prefix ...string) bool
+	substr(str string, start int, optionalEnd int)
+	to_lower(arg1 interface{}) interface{}
+	to_number(arg1 interface{}) interface{}
+	to_string(arg1 interface{}) interface{}
+	to_unix_time(input string, optionalLayout string) int64
+	to_upper(arg1 interface{}) interface{}
+	trim(arg1, arg2 interface{}) interface{}
+	trim_left(arg1, arg2 interface{}) interface{}
+	trim_prefix(arg1, arg2 interface{}) interface{}
+	trim_right(arg1, arg2 interface{}) interface{}
+	trim_space(arg1 interface{}) interface{}
+	trim_suffix(arg1, arg2 interface{}) interface{}
+	uniq(args ...interface{}) interface{}
+	unix_time(optionalSeconds uint) float64
+	url_decode(arg1 interface{}) interface{}
+	url_encode(arg1 interface{}) interface{}
+	wait_for(seconds uint)
+	zlib(arg1 interface{}) interface{}
+	zlib_decode(arg1 interface{}) interface{}
 `
-	t.Run("with coloring", func(t *testing.T) {
-		coloredSignatures := GetPrintableDslFunctionSignatures(false)
-		assert.Equal(t, expected, coloredSignatures)
-	})
-
-	t.Run("without coloring", func(t *testing.T) {
-		var decolorizerRegex = regexp.MustCompile(`\x1B\[[0-9;]*[a-zA-Z]`)
-		expectedSignaturesWithoutColor := decolorizerRegex.ReplaceAllString(expected, "")
 
-		assert.Equal(t, expectedSignaturesWithoutColor, GetPrintableDslFunctionSignatures(true))
-	})
+	signatures := GetPrintableDslFunctionSignatures(true)
```

Comment:
Even if we've decided to remove the color codes from the test itself it doesn't mean we shouldn't have some assertions on the colored signatures. E.g. they start with a specific ANSI color code, end with another before the first parenthesis and so on. Or we can at least take one specific signature and at least assert that it is present in the list to make sure that the coloring works as expected.